darts/metrics/__init__.py

```diff
@@ -14,6 +14,7 @@
     ope,
     r2_score,
     rho_risk,
+    quantile_loss,
```

Comment:
Hi,

I think that this line is causing issues with darts linter verification. Can you try to follow the instructions found [here](https://github.com/unit8co/darts/blob/master/CONTRIBUTING.md) to install the dev tools so that the tests can pass.

Reply:
Hi, thanks for the review, I've tried to install the pre-commit hooks for a while, but I either get 
```
RuntimeError: The Poetry configuration is invalid:
        - [extras.pipfile_deprecated_finder.2] 'pip-shims<=0.3.4' does not match '^[a-zA-Z-_.0-9]+$
```
error when running it or have to use `pre-commit autoupdate`  which changes pre-commit-config. I can't find anything new on github or SO, if it's something you dealt with before I'd really appreciate the help, if not no biggie and I'll just  try to remember to run black and isort manually before PRs